### DISCUSSION TITLE
What to do if the Chrome Web Store is blocked

### DISCUSSION THREAD

**Discussion by Loganrocks86** [3kh0/ext-remover#33] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
chrome webstore is blocked is there a different link?

**Comment by 3kh0** [comment, marked as answer]
I do not know of any alternate links out there, but I think this might work. Can you try adding something to the end of https://chrome.google.com/webstore and see if that will work? Example: https://chrome.google.com/webstoreAAAAAAAAAAA If this does not work, then there are no other solutions that I know of.

**Reply by Loganrocks86** [reply]
it says that chrome.google.com is blocked when i type that in just like the webstore

**Reply by 3kh0** [reply]
@Loganrocks86 did you try the second link, if that does not work then nothing can

**Reply by Demord64** [reply]
THX A LOT THE SECOND ONE WORKS

**Reply by Alex2Mayhem** [reply]
This doesn't work please find a webstore alttttt please.  it   has to do with this chrome policy 
![image](https://user-images.githubusercontent.com/104642381/197006942-74f9001c-e28f-4724-99cc-d1abcbd23e7d.png)

**Reply by amelia-bartlett** [reply]
can you open [this](https://chrome.google.com/webstore/detail/ublock-origin/cjpalhdlnbpafiamejdnhcphjbkeiagm) @Alex2Mayhem

**Reply by Aka-but-nice** [reply]
It seems as he also has JavaScript urls disabled too, I would recommend trying Killcurly, which doesn't bypass webstore but may bypass other things, considering your blocker

**Reply by 3kh0** [reply]
@Alex2Mayhem nice it department, aka pointed out some solutions, one more i will add is if you really want to, just unenroll, having to preform the exploits over and over because of updates is silly

**Reply by amelia-bartlett** [reply]
> can you open [this](https://chrome.google.com/webstore/detail/ublock-origin/cjpalhdlnbpafiamejdnhcphjbkeiagm)
tested on my own device, it won't work :(

**Comment by Alex2Mayhem** [comment]
im sorry what are we talking about?. im replying from email because my
school restricted my whole chromebook.

On Thu, Feb 2, 2023 at 9:13 PM Echo ***@***.***> wrote:

> @Alex2Mayhem <https://github.com/Alex2Mayhem> nice it department, aka
> pointed out some solutions, one more i will add is if you really want to,
> just unenroll, having to preform the exploits over and over because of
> updates is silly
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/3kh0/ext-remover/discussions/33#discussioncomment-4857542>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AY6LOTLTBYDLYYYRPOVXUF3WVRSTXANCNFSM6AAAAAAQ545KIU>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
(Owner) - Midnight Gaming Network - Fx
 🛡️️Staff Response Team🛡️

**Comment by amelia-bartlett** [comment]
do u have access to another device?  i'd recommend not using chrome and using firefox on it to prevent the school management from getting onto it (if you have admin on it you can also use registry patches (windows) or json config files (linux) or plist files (os x) to disable it but its more effort)
if u dont have admin on another device see if you can get hold of a linux live usb/dvd ([tails](https://tails.boum.org/) is good for bypassing firewalls and gaining privacy) from someone else who can flash you one, and boot it.  it wont work on chrome os without enabling dev mode and usb boot tho.

im assuming you are on a chrome book, if this is the case, then sh1mmer might be your best option.  if you are not, you could try a direct usb boot (if there is a firmware password, try [this](https://bios-pw.org/)) with [tails](https://tails.boum.org/) or something (you can also use [any os that can be made to run like this](https://www.debian.org/CD/live/))
